clocksilencer.ino

```diff
@@ -40,22 +35,28 @@ void setup() {
     while (rtc.getYear() == 1970) {
       Serial.println("RTC updating in progress");
     }
+    Serial.println("RTC update done");
+    rtcTimeWasSetFromNTP=true; //time was updated
     WiFi.disconnect();
     WiFi.mode(WIFI_OFF);  //disconnect after synchronisation to save power
     Serial.println("WiFi turned off");
     //play chime when everything ready
-    myDFPlayer.volume(5);  //Set volume value. From 0 to 30
-    myDFPlayer.play(1);
+    //mp3.volume(15);  //Set volume value. From 0 to 30
+    mp3.playbackSource(2);
```

Comment:
There is a [constsant for TF card](https://github.com/PowerBroker2/DFPlayerMini_Fast/blob/master/src/DFPlayerMini_Fast.h#L106), so this could be:

```suggestion
    mp3.playbackSource(TF);
```

Reply:
Afaik the numbers are also allowed. But we can try with that